wagtail/compat.py

```diff
@@ -27,3 +29,16 @@ class HTTPMethod(Enum):
         PUT = "PUT"
         DELETE = "DELETE"
         PATCH = "PATCH"
+
+
+URLField = forms.URLField
+
+# Prevent deprecation warning about the default scheme changing from "http" to
+# "https" in Django 5.0, while also avoiding the need to set the
+# FORMS_URLFIELD_ASSUME_HTTPS that would raise a different deprecation warning.
+if django.VERSION >= (5, 0) and django.VERSION < (6, 0):
+
+    class URLField(forms.URLField):
```

Comment:
Not using the `partial()` trick from [Adam Johnson's blog post](https://adamj.eu/tech/2023/12/07/django-fix-urlfield-assume-scheme-warnings/#third-party-packages) because it cannot be used for subclassing, which we need for `URLOrAbsolutePathField`.